### PR TITLE
Makes Generic Cola Not Hidden, Off-Brand Cola Hidden

### PR DESCRIPTION
**Pull request by Wisemonster**
[BALANCE] [CATERING] [GAME OBJECTS]

## About the PR 
This pr swaps the generic cola and off-brand colas hidden statuses, making it so the off-brand cola requires hacking the vendors to access, but the generics colas are already available. Also adjusts quantities to account for this.

## Why's this needed? 
It's a bit weird that the off-brand, random chemical soda is available by default, rather than the generic soda.

## Changelog 

```changelog
(u)Wisemonster
(+)The generic space cola is no longer hidden in soda vending machines, but the off-brand colas now require hacking the vendor to access
```

### CODE REVIEW

**Comment by TDHooligan** [comment]
given neither are harmful (and off-brands are a little more interesting) i'd rather have it as it is or have both

**Comment by Wisemonster** [comment]
> given neither are harmful (and off-brands are a little more interesting) i'd rather have it as it is or have both

Define harmful? Cause I know for sure some drinks will spawn things like mercury or other harmful-to-drink chems.

**Review by Tarmunora — APPROVED** [review]

**Comment by TDHooligan** [comment]
> Define harmful? Cause I know for sure some drinks will spawn things like mercury or other harmful-to-drink chems.

the pr's already passed but random cola uses flavors defined in COLA_flavors @
https://github.com/goonstation/goonstation/blob/master/strings/chemistry_tools.txt

File: code/modules/vending/vending.dm [hunk 1]
```diff
@@ -1460,7 +1460,7 @@ ABSTRACT_TYPE(/obj/machinery/vending/cola)
 			product_list += new/datum/data/vending_product(/obj/item/reagent_containers/food/drinks/bottle/soda/lime, 10, cost=PAY_UNTRAINED/6)
 			product_list += new/datum/data/vending_product(/obj/item/reagent_containers/food/drinks/bottle/soda/grones, 10, cost=PAY_UNTRAINED/6)
 			product_list += new/datum/data/vending_product(/obj/item/reagent_containers/food/drinks/bottle/soda/bottledwater, 10, cost=PAY_UNTRAINED/4)
-			product_list += new/datum/data/vending_product(/obj/item/reagent_containers/food/drinks/cola/random, 10, cost=PAY_UNTRAINED/10) //does this even work??
+			product_list += new/datum/data/vending_product(/obj/item/reagent_containers/food/drinks/cola/random, rand(1, 5), cost=PAY_UNTRAINED/10, hidden = 1) //does this even work??
```
Comment (ZeWaka) [inline code/modules/vending/vending.dm:1463]:
```suggestion
			product_list += new/datum/data/vending_product(/obj/item/reagent_containers/food/drinks/cola/random, rand(1, 5), cost=PAY_UNTRAINED/10, hidden = 1)
```

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**